frontend/src/app/components/key-result-form/key-result-form.component.html

```diff
@@ -15,11 +15,7 @@
     </div>
   </div>
 
-  <app-keyresult-type
-    (formValidityEmitter)="updateFormValidity()"
```

Comment:
why is the formValidityEmitter removed?

Reply:
i removed the input from key-result-type-component because it was unused